src/auro-radio-group.js

```diff
@@ -109,13 +156,13 @@ export class AuroRadioGroup extends LitElement {
    * @returns {void}
    */
   handleSelection(event) {
-    if (event.target.hasAttribute('value')) {
+    if (event.target.value) {
```

Comment:
What happens here when the element is reset to the default state with no value selected?

Reply:
Not exactly sure what you are asking here? Though, this event/method only gets fired when a selection is made and the radio has the `checked` attribute. This change was made due to an issue it was causing in validation.

Reply:
After further discussion, this will be addressed as a part of a bigger update in this issue:
- https://github.com/AlaskaAirlines/auro-checkbox/issues/132